go/controller/autoscaler/cluster.go

```diff
@@ -147,7 +147,7 @@ func (c *K8sCluster) SyncResource() error {
 		}
 		for resname, q := range item.Status.Allocatable {
 			if resname == v1.ResourceCPU {
-				totalCPU += float64(q.Value()) + float64(q.MilliValue())/1000.0
```

Comment:
Go constant is very special:

> Go is a statically typed language that does not permit operations that mix numeric types. You can't add a float64 to an int, or even an int32 to an int. Yet it is legal to write 1e6*time.Second or math.Exp(1) or even 1<<('\t'+2.0). In Go, constants, unlike variables, behave pretty much like regular numbers. This post explains why that is and what it means.

So it's more idiomatic to simple use `1000` here, don't worry about accidentally mess up anything, the Go compiler is very strict about type, so if anything could go wrong, it won't compile.

Please see: https://blog.golang.org/constants